docs/docs/examples/guides/05-jtc-guide.md

```diff
@@ -0,0 +1,587 @@
+- `Command interfaces`: that refers to the type of command that JTC will use to control your robot; one of position,
+velocity, acceleration, or effort.
+- `State interfaces`: the type of state interfaces JTC will claim. Some interfaces may be necessary depending on your
+choice of command interfaces. For example, if you want to control your robot in acceleration, you need to have a
+velocity state interface. 
+- `Open loop control`: refers to whether the controller will be correcting its desired commands according to the state
+it is receiving or not.
+- `Allow nonzero velocity at trajectory end`: typically, you want the robot to halt when it reaches the end of your
+trajectory. However, there are cases, e.g., when you are controlling in velocities, where you want to allow non-zero
+values even if the trajectory waypoints have been successfully traversed.
+- `Proportional/Integral/Derivative gain(s)`: these gains are to be set and tuned when you are, for example, commanding
+in velocity but your trajectory has positions. Then, a PID control will be applied to generate desired velocities
+through the trajectory.
+
+You will also notice tolerance values for trajectory execution times and position offsets. These should be set on a
+per-application basis.
+
+You may find advanced parameters in the `Docs` tab under the `Joint Trajectory Controller` page. These parameters can be
+used to further tune JTC's performance and functionality, but may also require more advanced knowledge in order to tune
+them. Make sure to carefully read the descriptions next to each parameter before making changes.
+
+### Setting a trajectory for execution
+
+There are 2 ways of setting a trajectory in JTC:
+
+1. by using a `JointTrajectory` signal
+2. by calling the `set_trajectory` service
+
+In both cases, receiving a new joint trajectory will first trigger cancellation of the active (if any) trajectory. That
+is, **there is no trajectory buffering or appending taking place**. As with many things in the AICA Universe, behaviors
+are event-driven. If you wish to send multiple trajectories back-to-back, you will have to rely on the execution status
+of the active trajectory handled by JTC. There is a practical example of how do this in following sections
+(see [Putting an application together](#putting-an-application-together)). 
+
+#### Using JTC with signals
+
+In practice, this requires publishing a single
+[`JointTrajectory`](https://docs.ros.org/en/noetic/api/trajectory_msgs/html/msg/JointTrajectory.html) message. In AICA
+Studio terms, that means connecting an edge from a custom component directly to the input of JTC. If you already have a
+component that has the corresponding signal output, feel free to skip the next segment.
+
+If you are writing the signal logic yourself, the following code indicates what your message-related code may look like:
+
+<Tabs groupId="programming-language">
+<TabItem value="python" label="Python">
+
+```python title="py_component.py"
```

Comment:
Nice, is that tested?

Reply:
a few correction, it's now good and tested